maintainers/maintainer-list.nix

```diff
@@ -5881,6 +5881,12 @@
     githubId = 4579165;
     name = "Danny Bautista";
   };
+  peelz = {
+    email = "peelz.dev+nixpkgs@gmail.com";
+    github = "louistakepillz";
+    githubId = 920910;
+    name = "peelz";
+  };
```

Comment:
can you add this as part of another commit:
```
maintainers: add peelz
```
I should have caught this in my first pass, sorry